lib/poller.js

```diff
@@ -1,7 +1,7 @@
 const _ = require('lodash/fp')
 
 const plugins = require('./plugins')
-const notifier = require('./notifier')
+const notifier = require('./notifier/index')
```

Comment:
```suggestion
const notifier = require('./notifier')
```

Reply:
there are two notifier files, the one outside the notifier folder is the pre-refactor code. which is why I explicitly import the index.js file, to grab the new code.

Reply:
Why not remove the old one? We don't want to keep unused files around.